src/overlays/actors/ovl_Dm_Zl/z_dm_zl.h

```diff
@@ -8,12 +8,25 @@ struct DmZl;
 typedef void (*DmZlActionFunc)(struct DmZl*, GlobalContext*);
 
 typedef struct DmZl {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x11C];
-    /* 0x0260 */ DmZlActionFunc actionFunc;
-    /* 0x0264 */ char unk_264[0x70];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ UNK_TYPE1 pad188[0xD8]; // unused by DmZl
+    /* 0x260 */ DmZlActionFunc actionFunc;
+    /* 0x264 */ UNK_TYPE1 pad264[0x4C]; // unused by DmZl
+    /* 0x2B0 */ s16 animationIndex;
+    /* 0x2B2 */ u8 eyeTextureLeft;
+    /* 0x2B3 */ u8 eyeTextureRight;
+    /* 0x2B4 */ u8 mouthTextureIndex;
+    /* 0x2B5 */ u8 nextEyeState;   // used to control eye state, but not set by our actor, outside of actor?
+    /* 0x2B6 */ u8 nextMouthMouth; // used to control mouth state, but not set by our actor, outside of actor?
```

Comment:
Just noticed how weird this name is. Did you mean "nextMouthState"?

Reply:
oh yeah thats odd, probably was a search replace falsepositive.